### PR TITLE
Pass the current version + if we are on dev env on starting

### PR DESCRIPTION
**Pull request by RodrigoHolztrattner**
Requires this frontend PR: https://github.com/stream-labs/streamlabs-obs/pull/1694

### CODE REVIEW

File: crash-handler-process/metricsprovider.hpp [hunk 1]
```diff
@@ -55,11 +55,12 @@ class MetricsProvider
     private:
 
     HANDLE        m_Pipe;
-    HANDLE        m_ClientConnectionThread = nullptr;
-    bool		  m_PipeActive = false;
-    bool          m_StopPolling = false;
+    HANDLE        m_ClientConnectionThread   = nullptr;
+    bool		  m_PipeActive               = false;
+    bool          m_StopPolling              = false;
```
Comment (eddyStreamlabs) [inline crash-handler-process/metricsprovider.hpp]: Style issues

File: crash-handler-process/metricsprovider.cpp [hunk 2]
```diff
@@ -393,6 +398,12 @@ std::string MetricsProvider::GetMetricsFileStatus()
 
 void MetricsProvider::SendMetricsReport(std::string status)
 {
+    // Disabled on dev env
+    if (m_IsDevEnv)
+    {
+        return;
+    }
```
Comment (eddyStreamlabs) [inline crash-handler-process/metricsprovider.cpp]: seems like the brackets are not needed here

File: crash-handler-process/main.cpp [hunk 3]
```diff
@@ -321,11 +321,19 @@ void restartApp(std::wstring path) {
...
+    std::string version;
+    std::string isDevEnv;
+
+    // Frontend pass as non-unicode
+	if (argc >= 1)
+		path = std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(argv[0]);
+    if (argc >= 3)
+        version = argv[2];
+    if (argc >= 4)
+        isDevEnv = argv[3];
```
Comment (eddyStreamlabs) [inline crash-handler-process/main.cpp]: style issues here

File: crash-handler-process/metricsprovider.hpp (hunk 1 above)
Comment (RodrigoHolztrattner) [reply crash-handler-process/metricsprovider.hpp]: Done

File: crash-handler-process/metricsprovider.cpp (hunk 2 above)
Comment (RodrigoHolztrattner) [reply crash-handler-process/metricsprovider.cpp]: Done

File: crash-handler-process/main.cpp (hunk 3 above)
Comment (RodrigoHolztrattner) [reply crash-handler-process/main.cpp]: Done

**Pull request merged**